wire-compiler/src/main/java/com/squareup/wire/OutputArtifact.java

```diff
@@ -0,0 +1,45 @@
+package com.squareup.wire;
+
+import java.io.File;
+
+/**
+ * A container class that represents an artifact output from the compiler.
+ */
+public final class OutputArtifact {
+  private final String outputDirectory;
+  private final String className;
+  private final String javaPackage;
+
+  public OutputArtifact(String outputDirectory, String javaPackage, String className) {
+    this.outputDirectory = outputDirectory;
+    this.className = className;
+    this.javaPackage = javaPackage;
+  }
+
+
+  public String outputDirectory() {
```

Comment:
@JakeWharton  did you mean for me to remove the get prefixes on the names on all of these methods?